packages/database/src/migrations/20201129051349-DropPSSSSessionNotifyChangesTrigger-modifies-schema.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql('DROP TRIGGER IF EXISTS psss_session_trigger ON psss_session;');
```

Comment:
When `psss_session` table was created, a change notify trigger was also created which was unncessary. The trigger also caused a message `payload string too long` whenever the session record is updated (mainly because we store the access_policy data which can be very long). So this is for dropping it

Reply:
Thanks Chris, great job working that one out and fixing it up!